modules/tools/00_mod.mk

```diff
@@ -262,6 +266,8 @@ GO_DEPENDENCIES += boilersuite=github.com/cert-manager/boilersuite
 GO_DEPENDENCIES += gomarkdoc=github.com/princjef/gomarkdoc/cmd/gomarkdoc
 GO_DEPENDENCIES += oras=oras.land/oras/cmd/oras
 GO_DEPENDENCIES += klone=github.com/cert-manager/klone
+GO_DEPENDENCIES += goreleaser=github.com/goreleaser/goreleaser
+GO_DEPENDENCIES += syft=github.com/anchore/syft/cmd/syft
```

Comment:
note: syft uses goreleaser for its own releases, and it defines variables in the linker which we'll leave unset since we're using go install. I think that's fine: https://github.com/anchore/syft/blob/0748413d6bd5f668a9c66ab5632d320df5e28458/.goreleaser.yaml#L27